openmc/filter.py

```diff
@@ -322,17 +323,17 @@ def get_bin_index(self, filter_bin):
 
             # Use lower energy bound to find index for energy Filters
             elif self.type in ['energy', 'energyout']:
-                val = self.bins.index(filter_bin[0])
+                val = np.where(self.bins == filter_bin[0])[0][0]
```

Comment:
I'm assuming this change is to support cases where self.bins is a numpy array rather than a list?